pnpm-workspace.yaml

```diff
@@ -1,8 +1,130 @@
 packages:
-  - 'projects/*/*'
+  - projects/*/*
   - '!projects/js-packages/jetpack-cli'
-  - 'projects/plugins/*/tests/e2e'
-  - 'tools/cli'
-  - 'tools/e2e-commons'
-  - 'tools/js-tools'
-  - '.github/files/coverage-munger'
+  - projects/plugins/*/tests/e2e
+  - tools/cli
+  - tools/e2e-commons
+  - tools/js-tools
+  - .github/files/coverage-munger
+
+engineStrict: true
+managePackageManagerVersions: false
+preferWorkspacePackages: true
+
+# Be strict about peer deps
+autoInstallPeers: false
+dedupePeerDependents: false
+strictPeerDependencies: true
+
+# No hoisting by default.
+hoistPattern: []
+
+# Silence this warning. Our `jetpack dependencies build-order` (also used by `jetpack build --all`) checks for cycles itself, plus it has a way to indicate that a dep is only for testing.
+ignoreWorkspaceCycles: true
+
+# Package compromise avoidance: let new releases age a day, and don't allow trust downgrades unless explicitly excepted.
```

Comment:
How will this work with Calypso packages? Does this just mean we need to explicitly state the update version?